lib/server.js

```diff
@@ -92,6 +92,22 @@ class Server {
     } = prepareApplications(applications));
   }
 
+  // Update application version for each connection.
+  // It will update to a newest version in the requested range. If client
+  // requested version in (1.1) range he will not be updated to (1.2.x). If no
```

Comment:
I think it'll be better to specify the first range as `(1.1.x)` instead of just `(1.1)` for consistency with the second one.
You should also add a comma after 'range': `'If client requested version in (1.1.x) range, he will...'`.